include/crypto/mlkem.h

```diff
@@ -27,201 +27,538 @@ typedef struct ossl_mlkem_ctx {
     char *properties;
 } ossl_mlkem_ctx;
 
+/* Variant-independent root of unity tables */
+extern const uint16_t ossl_mlkem_NTTRoots[128];
```

Comment:
Why would we have any declared global data like this?
None of this should be exposed or visible in a public header file - I don't get why they are here at all.

Reply:
This is obsolete, I'll delete this line.

Reply:
Pushed a fixup that also drops an unused variable and makes the "ModRoots" static, plus a couple of minor cosmetic tweaks.

Reply:
Thanks for reporting this, much appreciated.

Reply:
In terms of *installation*, this is an internal header file.  Only those in `include/openssl` are considered public.